.github/actions/buildkite/run.sh

```diff
@@ -76,6 +76,7 @@ echo "::endgroup::"
 
 URL=$(echo "$RESP" | jq -r ".url")
 WEB_URL=$(echo "$RESP" | jq -r ".web_url")
+echo "::notice title=Buildkite Build URL::${WEB_URL}"
```

Comment:
Do you know if the markdown link syntax works?

```suggestion
echo "::notice title=Buildkite Build URL::[${WEB_URL}](${WEB_URL})${WEB_URL}"
```

Reply:
I think it will be clickable by default.

https://github.com/orgs/community/discussions/47014

but I can run a quick test.

Reply:
maybe https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/ could be the alternative?

Reply:
I thought about the same..

But I was worried that.. if a workflow itself utilises the summary and we just append in some random place that the summary will look weird..

Reply:
yep, don't get crazy, I think your initial proposal is good enough!

Reply:
OH wow, the clickable link only works for github urls 😅

Reply:
![image](https://github.com/elastic/apm-pipeline-library/assets/16325797/9277b198-a57d-43f7-b9e8-470579f79833)

Reply:
I'm going to revert to https://github.com/elastic/apm-pipeline-library/pull/2498/commits/96a1f2ab1b9d4906c353b329c2296f6489143fd2